tensorflow/tools/api/golden/v1/tensorflow.pbtxt

```diff
@@ -336,10 +336,6 @@ tf_module {
     name: "errors"
     mtype: "<type \'module\'>"
   }
-  member {
-    name: "estimator"
-    mtype: "<type \'module\'>"
-  }
```

Comment:
What does `estimator` module has to do with MKL quantized ops?

Reply:
same as above, will remove it.